index.html

```diff
@@ -3319,7 +3319,15 @@ <h5>JWT Encoding</h5>
               </li>
               <li>
 <code>sub</code> MUST represent the <code>id</code> <a>property</a> contained
-in the <a>verifiable credential</a> <a>subject</a>.
+in the <a>verifiable credential</a> <a>subject</a>. Implementers are warned
```

Comment:
Could this be put into its own note?

Reply:
Done in 22a33da77b66c8ae6ca3040b2e2301bfd2e27f13.